azure-advanced/Packer/ansible-apim/dev/group_vars/apim-analytics.yml

```diff
@@ -0,0 +1,200 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2020 WSO2, Inc. http://www.wso2.org
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#----------------------------------------------------------------------------
+
+wso2_group: wso2
+wso2_user: wso2carbon
+product_name: wso2am-analytics
+product_version: 3.1.0
+target: /mnt
+product_package_location: files
+misc_file_location: "{{ product_package_location }}/misc"
+misc_file_copy_location: "{{ target }}/misc"
+security_file_location: "{{ product_package_location }}/security"
+jdbc_driver: mysql-connector-java-5.1.48-bin.jar
+
+# Performance tuning configurations
+enable_performance_tuning: false
+performance_tuning_file_list:
+  - { src: '{{ product_package_location }}/system/etc/sysctl.conf',
+      dest: '/etc/sysctl.conf' }
+  - { src: '{{ product_package_location }}/system/etc/security/limits.conf',
+      dest: '/etc/security/limits.conf' }
+
+# If custom keystores and trsustrores are need to be added, uncomment the below list
+# Add the required files under files/security/<product-home>/<path-to-file>
+# A sample is given below
+# security_file_list:
+#   - { src: '{{ security_file_location }}/wso2am-analytics/client-truststore.jks',
+#       dest: '{{ carbon_home }}/resources/security/client-truststore.jks' }
+#   - { src: '{{ security_file_location }}/wso2am-analytics/wso2carbon.jks',
+#       dest: '{{ carbon_home }}/resources/security/wso2carbon.jks' }
+
+# Uncomment and add any miscellaneous files you need to copy
+# A sample is given below
+# misc_file_list:
+#   - { src: '{{ misc_file_location }}/apimcli.zip',
+#       dest: '{{ misc_file_copy_location }}/apimcli.zip' }
+
+# Set the location the product packages should reside in (eg: "local" in the /files directory, "remote" in a remote location)
+pack_location: local
+#pack_location: remote
+#remote_jdk: "<URL_TO_JDK_FILE>"
+#remote_pack: "<URL_TO_APIM_ANALYTICS_PACK>"
+
+# JDK Distributions
+jdk_name: amazon-corretto-8.242.08.1-linux-x64
```

Comment:
@rasanjaya85 in other installation artifacts, the default JDK version packaged is `11`. Have we tested Amazon Corretto version 11 with WSO2 products?

Reply:
No. This version is use for ansible-apim releases and it's tested.

Reply:
Ack.

Reply:
@rasanjaya85 if so this needs to be fixed at Ansible level since, the default JDK for latest WSO2 products is JDK version `11`.

Reply:
Yes. +1 for create Git issue in the APIM Ansible git repository to track down.